linode/vpcsubnet/resource_test.go

```diff
@@ -31,7 +32,9 @@ func init() {
 		log.Fatal(fmt.Errorf("Error getting client: %s", err))
 	}
 
-	testRegion, err := acceptance.GetRandomRegionWithCaps([]string{"VPCs"})
+	testRegion := "us-east"
+	//testRegion, err := acceptance.GetRandomRegionWithCaps([]string{"VPCs"})
```

Comment:
We probably want to put it back to random region.

Reply:
oops fixed!